doc/Command-Reference.md

```diff
@@ -4108,6 +4108,37 @@ This command is used to delete the syslog server configured.
   Restarting rsyslog-config service...
   admin@str-s6000-acs-11:~$
   ```
+
+# DHCP Relay Destination IP address Configuration Commands 
+
+This sub-section of commands is used to add or remove the DHCP Relay Destination IP address on a VLAN interface.
```

Comment:
Minor rewording: `This sub-section of commands is used to add or remove DHCP Relay Destination IP address(es) for a VLAN interface.`